moveit_ros/moveit_servo/tests/test_ros_integration.cpp

```diff
@@ -103,6 +103,32 @@ TEST_F(ServoRosFixture, testJointJog)
 
   ASSERT_GE(traj_count_, NUM_COMMANDS);
 
+  // Ensure deprecation warning is reported for displacements command
+  jog_cmd.displacements.push_back(1.0);
+  jog_cmd.header.stamp = servo_test_node_->now();
+  joint_jog_publisher->publish(jog_cmd);
+  rclcpp::sleep_for(std::chrono::milliseconds(100));
+
+  // TODO how to test logged warnings?
```

Comment:
Not very easily unless you go down the whole dependency injection path... I would say this is quite an undertaking.

Reply:
Got this working by subscribing to logs on `/rosout`.